modules/intel/ifcore.f90

```diff
@@ -0,0 +1,22 @@
+! @author skosukhin
```

Comment:
```suggestion
!
! This file is released under terms of BSD license
! See LICENSE file for more information
!
! @author skosukhin
```

Reply:
CI is failing because the new module file is missing the license header

Reply:
@clementval I can add the header, sure. By the way, do you think we might get into any kind of conflict with Intel's licenses?

Reply:
@skosukhin No idea about the license but since it's just the signature I would guess that it should be ok ...